extension/content.js

```diff
@@ -268,7 +268,11 @@ function linkifyIssuesInTitles() {
 }
 
 function addPatchDiffLinks() {
-	const commitUrl = location.href.replace(/\/$/, '');
+	let commitUrl = location.href.replace(/\/$/, '');
+	const prCommit = /\/pull\/\d+\/commits/;
+	if (prCommit.test(commitUrl)) {
+		commitUrl = commitUrl.replace(prCommit, '/commit');
```

Comment:
Hmm, actually when checking it out, it seems GitHub copies the commits in the base repo so it's working properly even with PRs from forks.